### PR TITLE
Move Context Styles to Context Controller

### PR DESCRIPTION
**Pull request by bennypowers**
Removes the need to remember to apply context styles, just have to add the controller

### CODE REVIEW

File: elements/pfe-accordion/pfe-accordion-header.scss [hunk 1]
```diff
@@ -18,6 +18,8 @@
 
   box-sizing: border-box;
 
+  --context-dark-background-color: black;
+
```
Comment (heyMP) [inline elements/pfe-accordion/pfe-accordion-header.scss:22]: This was leftover from testing, right?

File: core/pfe-core/controllers/color-context-controller.ts [hunk 2]
```diff
@@ -1,12 +1,17 @@
...
 
 import { ContextEvent, createContext } from '../context.js';
 import { bound } from '../decorators/bound.js';
 import { Logger } from './logger.js';
+import { StyleController } from './style-controller.js';
+
+import CONTEXT_BASE_STYLES from './color-context-controller.scss';
 
 export class ColorContextController implements ReactiveController {
+  // static styles = CONTEXT_BASE_STYLES;
```
Comment (heyMP) [inline core/pfe-core/controllers/color-context-controller.ts:13]: Remove?

File: elements/pfe-accordion/pfe-accordion-header.scss (hunk 1 above)
Comment (bennypowers) [reply elements/pfe-accordion/pfe-accordion-header.scss:22]: probably

File: core/pfe-core/controllers/color-context-controller.ts (hunk 2 above)
Comment (bennypowers) [reply core/pfe-core/controllers/color-context-controller.ts:13]: yes

File: elements/pfe-accordion/pfe-accordion-header.scss (hunk 1 above)
Comment (bennypowers) [reply elements/pfe-accordion/pfe-accordion-header.scss]: removed

File: core/pfe-core/controllers/color-context-controller.ts (hunk 2 above)
Comment (bennypowers) [reply core/pfe-core/controllers/color-context-controller.ts]: removed

**Pull request merged**